pset-functions.js

```diff
@@ -135,11 +159,61 @@ Use the following test cases to confirm your program meets the success criteria
 console.log('Problem 3:')
 
 // Add your code below this line
+function scrabbleScore(word) {
+  // creates an array with each letter as a string
+  const letter = word.split("")
+  let letterOccurence = {}
+
+  //NEED to transfrom letters into lowercase
+  //goes through each element in the letter array
+  //assigns a value for the letter to a key in the letterValue object
+  letter.forEach((element) => {
+    if (letterOccurence[element] === undefined){
```

Comment:
@kyleligman interesting approach using an object to count the occurrences of the characters in the word and then using that object to help calculate the score.